extra_tests/snippets/stdlib_array.py

```diff
@@ -96,4 +97,10 @@ def test_array_frombytes():
 with assert_raises(IndexError):
     a[0] = 42
 with assert_raises(IndexError):
-    del a[42]
\ No newline at end of file
+    del a[42]
+
+test_str = '🌉abc🌐def🌉🌐'
+u = array('u', test_str)
+assert u.__reduce_ex__(1)[1][1] == list(test_str)
+assert str(loads(dumps(u, 1))) == f"array('u', '{test_str}')"
```

Comment:
```suggestion
assert str(loads(dumps(u, 1))) == f"array('u', '{test_str}')", str(loads(dumps(u, 1)))
```

then it will shows the value on assertion failure

Reply:
(_jedi mind trick wave_) [That was not the line you're looking for…](https://github.com/RustPython/RustPython/pull/3064/checks?check_run_id=3621875306#step:12:418)